crypto/evp/pmeth_lib.c

```diff
@@ -158,7 +158,7 @@ static EVP_PKEY_CTX *int_ctx_new(OPENSSL_CTX *libctx,
      * If the key doesn't contain anything legacy, then it must be provided,
      * so we extract the necessary information and use that.
```

Comment:
This comment could probably do with an update

Reply:
Done